.github/workflows/check.yaml

```diff
@@ -8,26 +8,27 @@ jobs:
   check:
     runs-on: ubuntu-latest
     steps:
-
       - name: Checkout
         uses: actions/checkout@v2
 
       - uses: subosito/flutter-action@v2
         with:
+          flutter-version: "3.0.x"
```

Comment:
Is there a reason not to leave this input unspecified, which according to [the docs for this action](https://github.com/subosito/flutter-action/blob/main/README.md#usage), should be the latest release for this channel?

Reply:
I think the step doesn't check major version changes - our runners did not get an update to Flutter 3, although it's released to stable channel for a while now, and the checks are still run with 2.10.5.

Besides, I think in that case, we may want to pin a major/minor versions on CI - if the runner would update to the latest stable version (3.0.0), all of our CI checks would start failing, so to avoid this, pinning the version seems to be a good idea